web/src/components/tasks/CheckElevatorRegistrationStatus.tsx

```diff
@@ -96,17 +102,16 @@ export const CheckElevatorRegistrationStatus = (props: Props): ReactElement => {
         </div>
         <div className="fdr flex-half">
           <div className="flex-half padding-right-1">
-            <label htmlFor="city">{Config.elevatorRegistrationSearchTask.zipCodeLabel}</label>
-            <TextField
-              value={formValues.zipCode}
-              onChange={handleChangeForKey("zipCode")}
-              variant="outlined"
-              inputProps={{
-                id: "zipcode",
-                "data-testid": "zipcode",
-                type: "number",
+            <label htmlFor="municipality">{Config.elevatorRegistrationSearchTask.municipalityLabel}</label>
+            <MunicipalityDropdown
+              fieldName={"municipalities"}
+              municipalities={formattedMunicipalities}
+              helperText={""}
```

Comment:
No helper text or validation logic because that is handled by the parent component (`ElevatorRegistrationTask`)